airbyte-protocol/models/src/main/resources/protocol_models/airbyte_schema.yaml

```diff
@@ -0,0 +1,58 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-protocol/models/src/main/resources/protocol_models/airbyte_schema.yaml
+title: AirbyteSchema
+description: Airbyte schema type definition
+type: object
+additionalProperties: false
+required:
+- streams
+properties:
+  streams:
+    type: array
+    items:
+      "$ref": airbyte_schema.yaml#/definitions/AirbyteStream
+  metadata:
+    "$ref": airbyte_metadata.yaml#/definitions/AirbyteMetadata
+
+definitions:
+
+  AirbyteStream:
+    type: object
+    additionalProperties: false
+    required:
+    - id
+    - name
+    - fields
+    properties:
+      id:
+        type: string
+        description: Stream's unique identifier.
+      name:
+        type: string
+        description: Stream's name.
+      fields:
+        type: array
+        items:
+          "$ref": airbyte_schema.yaml#/definitions/AirbyteField
+      metadata:
+        "$ref": airbyte_metadata.yaml#/definitions/AirbyteMetadata
+
+  AirbyteField:
+    type: object
+    additionalProperties: false
+    required:
+    - id
+    - name
+    - type
+    properties:
+      id:
+        type: string
+        description: Field's unique identifier.
+      name:
+        type: string
+        description: Field's name.
+      type:
+        "$ref": airbyte_types.yaml
```

Comment:
Should this reference `AirbyteType` directly?

After the sync today will we codify `object` here or as the only allowable `AirbyteType`?

Reply:
hum actually the schema is a bit harder to describe here. I need to think about it.